homeassistant/components/websocket_api/commands.py

```diff
@@ -504,7 +504,7 @@ def _cached_template(template_str: str, hass: HomeAssistant) -> template.Templat
         vol.Optional("entity_ids"): cv.entity_ids,
         vol.Optional("variables"): dict,
         vol.Optional("timeout"): vol.Coerce(float),
-        vol.Optional("strict", default=False): bool,
+        vol.Optional("report_errors", default=False): bool,
```

Comment:
This will also require a frontend update.

Do we know any other places that use this API option?

Reply:
Yes, there are other users. I'll rework the PR so `strict` mode is still supported.

Reply:
Fixed